Rocket.toml

```diff
@@ -1,8 +1,14 @@
+[global.limits]
+forms = 1048576
+json  = 1048576
+
 [development]
 template_dir = "templates/"
 
 [staging]
 template_dir = "templates/"
 
 [production]
+port = 8000
 template_dir = "templates/"
+secret_key = "PswVQIoG2GUAR4AE2lSwsjGpzM4Bz8ppl8Apt4Z7C7E="
```

Comment:
this was never a real key, only used on my localhost, but i'll yank it out of here